curriculum/challenges/english/blocks/lecture-working-with-data-in-react/673500bfe1f41601c1ddb1a2.md

```diff
@@ -97,39 +100,33 @@ Remember what we said about the naming convention for CSS properties in React in
 `"background-color"`
 
 ### --feedback--
-
 Remember what we said about the naming convention for CSS properties in React inline styles.
 
 ---
 
 `background_color`
 
 ### --feedback--
-
```

Comment:
you have removed a lot of empty lines, please don't remove the empty lines, the markdown files are precisely structured and we want to keep the precise structure